accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
+                </li>
+                <li>Both the Accessibility Assessment view and the Accessibility Checker view are fully keyboard accessible, navigate as follows:
+                </li>
+                  <ul
+                    style={{
+                      listStyleType: "circle",
+                      marginInlineStart: "2rem",
+                    }}
+                  >
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Use the <strong>'tab'</strong> key to navigate to any focusable element in the checker, starting
+                        with the <strong>'Scan'</strong> button once the checker launches.
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        After running the scan, press the <strong>'tab'</strong> key again, to navigate to the <strong>Reset Selections</strong> icon and the <strong>Report</strong> icon.
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to navigate to the checkbox by each issue type and press the <strong>'Enter'</strong> key to filter the list of issue by <strong>Violations</strong>, <strong>Needs review</strong> and/or by <strong>Recommendations</strong>.
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to navigate to the <strong>Issue List</strong> tabs and use the <strong>'right'</strong> or the <strong>'left'</strong> arrow keys to navigate between the <strong>Element Roles</strong> view, the <strong>Checkpoints</strong> view and the <strong>Rules</strong> view.
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to navigate to the <strong>issue grouping</strong> and use the <strong>'enter'</strong> to open or close the issue grouping. Press the <strong>'tab'</strong> key to navigate to each issue, and press the <strong>'enter'</strong> key to select any issue.
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to move to the <strong>Learn more</strong> link or to move to the next issue.
+                      </p>
+                    </li>
+                  </ul>  
+                <li>
+                  Use the <strong>'headings'</strong> hierarchy or the implemented <strong>'landmarks'</strong> to quickly navigate from one section to another. The list of implemented landmarks are as follows:
```

Comment:
Headings and landmarks are not part of the user interface text, so shouldn't have quotes.